README.md

```diff
@@ -51,7 +53,7 @@ The `.nodeshift` directory contains your resource fragments.  These are `.yml` f
 
 OpenShift resource fragments are user provided YAML files which describe and enhance your deployed resources.  They are enriched with metadata, labels and more by nodeshift.
 
-Each resource gets its own file, which contains some skeleton of a resource description. Nodeshift will enrich it and then combine all the resources into a single openshift.yml and openshift.json(located in ./tmp/nodeshift/resource/).
+Each resource gets its own file, which contains some skeleton of a resource description. Nodeshift will enrich it and then combine all the resources into a single openshift.yml and OpenShift.json(located in ./tmp/nodeshift/resource/).
```

Comment:
Is this meant to have changed case?

Reply:
this is a file name, so it shouldn't change case

Reply:
I can revert, that was due quick small change to trigger other build to avoid circle CI running job again.
Thanks for the review.

Reply:
thanks for the review :+1:

Reply:
@lholmquist @richardlau filename reverted :+1: